modules/store/src/state.ts

```diff
@@ -4,20 +4,20 @@ import { Observable } from 'rxjs/Observable';
 import { Subscription } from 'rxjs/Subscription';
 import { queue } from 'rxjs/scheduler/queue';
 import { observeOn } from 'rxjs/operator/observeOn';
-import { startWith } from 'rxjs/operator/startWith';
 import { withLatestFrom } from 'rxjs/operator/withLatestFrom';
 import { scan } from 'rxjs/operator/scan';
 import { ActionsSubject } from './actions_subject';
 import { Action, ActionReducer } from './models';
 import { INITIAL_STATE } from './tokens';
 import { ReducerObservable } from './reducer_manager';
 import { ScannedActionsSubject } from './scanned_actions_subject';
+import { INIT } from './';
```

Comment:
Haha, got it. It took me an embarrassingly long time to figure out what you were referring to.